crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Effect.cs

```diff
@@ -57,21 +86,28 @@ void OnPreRenderUnderwaterEffect()
 
             RenderTextureDescriptor descriptor = XRHelpers.GetRenderTextureDescriptor(_camera);
             descriptor.useDynamicScale = _camera.allowDynamicResolution;
+            // PERF: Auto resolves which triggers more resolves than necessary.
```

Comment:
could you expand this comment. is this a todo or is it more a note that its not ideal? can this cost be avoided in some cases or does it always need to be paid?

Reply:
I have solved the issues in the comment and removed. Some code there was leftovers from trying to get CopyTexture and MSAA working which I think is a Unity bug. Going to look again at getting CopyTexture and MSAA working in 2021.2.